index.md

```diff
@@ -195,96 +129,113 @@ either the result of a digital migration or information that enables the creatio
-by the CSIP. Against this background, the AIP format will be introduced as a
-container format which allows managing the life-cycle of an E-ARK IP that
-starts with the ingest of an SIP.
+From the point of view of preserving the integrity of the AIP, the ideal case 
+isthat the logical AIP is packaged as one single physical container, because 
+all themetadata and content required to interprete the information package is 
+available inone single entity. In reality, however, this is not always possible 
+because the size of the physical container can become very large.
 
-As already mentioned in section [4.1](#reps), the CSIP relies on the concept of
-"representations". Figure [1](#fig1) gives an example of the structure of an IP
-with two representations.
+For this reason, the AIP format describes how to partition the AIP and keep 
+representations or representation parts in separate physical container files 
+(see section [5.1](#compdiv)). Even though this puts the integrity of the AIP 
+at risk - because in case of disaster recovery the physical container does not 
+represent the complete intellectual entity and dependencies to another (lost) 
+physical container can potentially make it impossible to interpret, understand, 
+or render the content - it is a necessary measure if the amount of data exceeds 
+the capacity limitation of long-term storage media.
 
-<a name="fig1"></a>
-![Information Package structure](figs/visio/fig_1_ip_structure.svg "Structure of an IP compliant with the E-ARK Common Specification for IPs structure")
+*Definition:* *Segmentation* is a physical manifestation of a logical AIP where a set of 
+physical container files contains parts of the logical AIP. Each segment of the logical AIP 
+is a packaged as a TAR or ZIP file and contains its own structural metadata.
 
-**Figure 1:**
-Structure of an IP.
+When the AIP is segmented, the structure of the segmentation needs to be recorded in the
+structural metadata.
 
-Furthermore, Figure [1](#fig1) shows that metadata can be stored either at the
-representation level or at the package level. In the CSIP it is explained in more
-detail where different types of metadata can be stored. In this context, it is
-sufficient to mention that descriptive, technical, preservation, and rights
-metadata can relate either to the IP as a whole or to individual representations.
+In [@OAIS2012] p. 1-9, the Archival Information Collection (AIC) is described as  
+“an Archival Information Package whose Content Information is an aggregation of other 
+Archival Information Packages." The AIC can therefore represent a the structure of a segmented 
+AIP is defined by a header information package (AIC) pointing to the child information 
+packages (AIPs). 
 
-### Representations in the AIP
+It is recommended that the structural metadata of the child information packages record
+to which header information package (AIC) they belong. 
```

Comment:
This is a two way pointer, e.g. a "double link" from parent to child and from child to parent?

Reply:
Yes. However, when you do a migration with many childs, you do not need to update the references of the childs. At least you know to which head package a child that was not concerned by a change - originally belonged.